docs/design.md

```diff
@@ -149,7 +149,8 @@ The `Logic` component handles the business logic of TEAMMATES. In particular, it
 
 Package overview:
 - **`logic.api`**: Provides the API of the component to be accessed by the UI.
-- **`logic.core`**: Contains the core logic of the system.
+- **`logic.core`**: Contains the core logic of the system, mainly CRUD of storage entities.
```

Comment:
Logic is not just CRUD; CRUD is the duty of storage layer. What is it that makes the logic layer "one step further"?

Reply:
I think it provides cascaded logic for CRUD (relationships between entities) and also sanitizing for UI input. These are also mentioned above so I guess it should be fine not to mention it again. Changes were made accordingly.